src/projections.jl

```diff
@@ -84,30 +84,59 @@ lat2colat, colat2lat
 ################################################################################
 
 """
-    function equiprojinv(x, y)
+    equiproj(lat, lon)
+
+Equirectangular projection. Given the latitude `lat` (in radians) and the
+longitude (in radians), return a tuple (Bool, Number, Number) where the
+first Boolean is a flag telling if the point falls within the projection (true)
+or not (false), and the two numbers are the x and y coordinates of the point
+on the projection plane (both are in the range [−1, 1]).
+"""
+function equiproj(lat, lon)
+    # The weird usage of mod2pi(x + π) - π allows to have angles in the range
```

Comment:
Wouldn't `rem2pi(x, RoundNearest)` be equivalent?

Reply:
I did not realize there was this nice solution. Thanks!